massgen/tests/test_bootstrap_criteria.py

```
@@ -0,0 +1,476 @@
+        orch = self._make_full_orch("bootstrap_inline")
+        # Round 1: each agent emits a proposal via submit_checklist handler
+        # (simulated here by populating the AgentState buffer directly).
+        orch.agent_states["a1"].criteria_proposals = [
+            {"text": "Concrete examples over abstractions", "category": "primary"},
+        ]
+        orch.agent_states["a2"].criteria_proposals = [
+            {"text": "State assumptions explicitly", "category": "standard"},
+        ]
+
+        # Round 2: orchestrator resolves criteria for the next prompt build.
+        items, cats, _vby, source, _anti, _anchors = orch._resolve_effective_checklist_criteria("a1")
```

Ruff (0.15.12)

[warning] Unpacked variable `cats` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)